modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -142,6 +142,9 @@ The query service provides an array of options to customize your query. The foll
 | `ScanConsistency QueryScanConsistency` | Sets a different scan consistency for this query.
 |====
 
+NOTE: Unlike other SDKs the Go SDK defaults to sending queries as https://docs.couchbase.com/server/current/guides/prep-statements.html[prepared statements].
```

Comment:
We'll put this through just here for now, but may want to rewrite this page at some time, and find a better opportunity to talk about adhoc.